installer_show_maps_data/prep_data_for_maps.py

```diff
@@ -0,0 +1,315 @@
+# %% [markdown]
+# ## Preparing data for installerSHOW maps and plots
+#
+# This notebook is used to prepare the data for the installerSHOW maps and plots data story.
+#
+# The plots are visualised using Flourish.
+
+# %%
+import pandas as pd
+
+# %% [markdown]
+# ## Loading MCS installations data
+
+# %%
+installations_data = pd.read_csv(
+    "s3://asf-core-data/outputs/MCS/mcs_installations_240415.csv"
+)
+
+# %% [markdown]
+# ## Loading local authority and dealing with missing local authorities
+
+# %%
+# information about local authorities from Flourish
+flourish_LAs = pd.read_csv(
+    "s3://asf-exploration/installer_show_maps_data/inputs/flourish_LAs_and_codes.csv"
+)
+
+# %%
+# information about local authorities from MCS
+mcs_LAs = pd.DataFrame(installations_data["local_authority"].unique()).rename(
+    columns={0: "Name"}
+)
+mcs_LAs = mcs_LAs[~pd.isnull(mcs_LAs["Name"])]
+
+
+# %%
+len(flourish_LAs), len(mcs_LAs)
+
+# %%
+# saving MCS local authorities that are not in Flourish to S3
+mcs_LAs[~mcs_LAs["Name"].isin(flourish_LAs["Name"])].to_csv(
+    "s3://asf-exploration/installer_show_maps_data/inputs/missing_LAs.csv", index=False
+)
+
+# %%
+# saving Flourish local authorities that are not in MCS
+flourish_LAs[~flourish_LAs["Name"].isin(mcs_LAs["Name"])]
+
+# %% [markdown]
+# ### Updating local authority information
+
+# %%
+# we've mapped MCS LAs to the Flourish LA
+missing_LAs_update = pd.read_csv(
+    "s3://asf-exploration/installer_show_maps_data/inputs/missing_LAs_updated.csv"
+)
```

Comment:
I did the mapping manually yes, I'll update the comment. I think it's pretty straightforward, I did it by checking on google. Don't want to waste your time, but equally if you're happy to review, then it would be better. It's 40 manual checks though.